theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/DopeSheetSelectionView.tsx

```diff
@@ -50,13 +52,22 @@ const DopeSheetSelectionView: React.FC<{
   )
 }
 
+/**
+ * The horizontal and vertical bounds of the selection, each represented by a tuple in the form of [from, to].
+ */
 type SelectionBounds = {
-  positions: [from: number, to: number]
-  ys: [from: number, to: number]
+  /**
+   * The horizontal bounds of the selection as a tuple of "from" and "to" coordinates, "from" representing the start of the drag.
+   */
+  h: [from: number, to: number]
+  /**
+   * The vertical bounds of the selection as a tuple of "from" and "to" coordinates, "from" representing the start of the drag.
+   */
+  v: [from: number, to: number]
 }
 
 function useCaptureSelection(
-  layoutP: Pointer<SequenceEditorPanelLayout>,
+  layout: SequenceEditorPanelLayout,
```

Comment:
using `layout` instead of `layoutP` is going to make this hook re-evaluate its `useMemo` very time `layout` changes when the selecting drag gesture is not running.